l10n_it_bill_of_entry/__manifest__.py

```diff
@@ -9,16 +9,13 @@
 
 {
     "name": "ITA - Bolle doganali",
-    "version": "16.0.1.0.1",
+    "version": "18.0.1.0.0",
     "category": "Localization/Italy",
     "author": "Agile Business Group, CQ Creativi Quadrati, TAKOBI, "
     "Odoo Community Association (OCA)",
-    "website": "https://github.com/OCA/l10n-italy" "/tree/16.0/l10n_it_bill_of_entry",
+    "website": "https://github.com/OCA/l10n-italy",
     "license": "AGPL-3",
-    "depends": [
-        "base",
-        "account",
-    ],
+    "depends": ["base", "account", "l10n_it"],
```

Comment:
La dipendenza da `l10n_it` in passato andava evitata, ma direi che ormai l'abbiamo sdoganata :wink: (pun intended)
:+1: